src/Bridge/Symfony/XliffDumper.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\MessageCatalogue;
+
+final class XliffDumper extends XliffFileDumper
```

Comment:
Why is this class needed? Couldn't we use `Translation\SymfonyStorage\XliffDumper`?